config/management/Cargo.toml

```diff
@@ -20,6 +20,7 @@ libra-crypto = { path = "../../crypto/crypto", version = "0.1.0" }
 libra-network-address = { path = "../../network/network-address", version = "0.1.0" }
 libra-secure-storage = { path = "../../secure/storage", version = "0.1.0" }
 libra-secure-time = { path = "../../secure/time", version = "0.1.0" }
+libra-shared-constants = { path = "../../config/shared-constants", version = "0.1.0"}
```

Comment:
maybe just constants or global constants?

Reply:
Went for "global constants"